src/nvim/os/users.c

```diff
@@ -41,7 +41,11 @@ int os_get_usernames(garray_T *users)
 // Return OK if a name found.
 int os_get_user_name(char *s, size_t len)
 {
+#ifdef UNIX
   return os_get_uname(getuid(), s, len);
+#else
+  return os_get_uname(0, s, len);
```

Comment:
Both I guess. Its workable in that it should work fine for anything non-Unix, FAIL is a valid return for that function.

Ideally, for Windows, we would implement something better but I don't have a working solution yet.